src/exporters/srtm.py

```diff
@@ -0,0 +1,68 @@
+import elevation
+from osgeo import gdal
+from pathlib import Path
+
+from typing import Tuple
+
+from .base import BaseExporter
+from ..utils import Region, region_lookup
+
+
+class SRTMExporter(BaseExporter):
+    """Export SRTM elevation data. This exporter leverages
+    the elevation package, http://elevation.bopen.eu/en/stable/, to download
+    SRTM topography data.
+    """
+
+    dataset = 'srtm'
+
+    @staticmethod
+    def _region_to_tuple(region: Region) -> Tuple[float, float, float, float]:
+        return region.lonmin, region.latmin, region.lonmax, region.latmax
+
+    @staticmethod
+    def _tiff_to_nc(tif_file: Path, nc_file: Path) -> None:
```

Comment:
WOHOOOO this is perfect because working with `tiff` files can be a pain in the booty

Reply:
any reason why `tiff` vs. `tif`?

Reply:
no haha - arbitrary
I guess it would be more consistent to have either `tif_to_nc` or `tiff_to_netcdf` - I'll make that quick change